### DISCUSSION TITLE
Add an option to disable the sorting functionality in the target list

### DISCUSSION THREAD

**Discussion by Lazerproof** [formkit/formkit#1406] [Ideas]
I have a use case where I have a fixed sort order of the source options and I want the same sort order in target values without allowing the user to change the sort. I assume this is already possible by overriding some field slots (is it?), but I would love to see a simpler implementation by prop setting.

**Comment by sashamilenkovic** [comment]
@Lazerproof Not sure if that's going to be easy to do via. overriding slots. The transferlist wasn't really built with what you're suggesting in mind. Perhaps we could add a prop for this in the future. For your usecase, it might be best to simply use an input hook and and manually sort the values. Using a plugin you assign to the transferlist like so:

```
function sortValues(node) {
  node.hook.input((payload, next) => {
    if (payload?.length) {
     // Sort however you need
      payload = payload.sort()
    }
    return next(payload)
  })
}
```

Do you think something like this would work for your use-case?
